demonstrations/tutorial_differentiable_HF.py

```diff
@@ -0,0 +1,389 @@
+r"""
+
+Differentiable Hartree-Fock
+===========================
+
+.. meta::
+    :property="og:description": Learn how to use the differentiable Hartree-Fock solver
+    :property="og:image": https://pennylane.ai/qml/_images/differentiable_HF.png
+
+.. related::
+    tutorial_quantum_chemistry Quantum chemistry with PennyLane
+    tutorial_vqe A brief overview of VQE
+    tutorial_givens_rotations Givens rotations for quantum chemistry
+    tutorial_adaptive_circuits Adaptive circuits for quantum chemistry
+
+
+*Author: PennyLane dev team. Posted:  2022. Last updated: 21 April 2022*
+
+In this tutorial, you will learn how to use PennyLane's differentiable Hartree-Fock solver
+[#arrazola2021]_. The Hartree-Fock module in PennyLane provides built-in methods for constructing

```

Comment:
```suggestion
[#arrazola2021]_. The quantum chemistry module in PennyLane provides built-in methods for constructing
```
Maybe good to add a link to it too